app/Http/Controllers/Api/LinkController.php

```
@@ -44,10 +44,10 @@ public function store(Request $request)
 
         $url = $request->input('url');
         $host = parse_url($url, PHP_URL_HOST);
-        if (in_array($host, $whitelist)) {
-            $url = $this->bertly->shorten($url);
+        if (! in_array($host, $whitelist)) {
+            return ['url' => $url, 'count' => '0'];
```

Eventually, do you think we'd want to return an error of some sort or `404` in this scenario?

---

Yeah, that might be better. Wasn't sure how strict we wanted to be.